SQL/0000-00-00-schema.sql

```diff
@@ -1040,6 +1040,7 @@ CREATE TABLE `session` (
   `MRIQCPending` enum('Y','N') NOT NULL default 'N',
   `MRIQCFirstChangeTime` datetime default NULL,
   `MRIQCLastChangeTime` datetime default NULL,
+  `Caveat` enum('true','false') NOT NULL default 'false',
```

Comment:
Is this Caveat flag for the whole session, or just for the imaging?

It's on the session table, so it's a little confusing that it's called "session.Caveat" but referred to as "mricaveat" in the template.